modules/Spoke/alerts.tf

```diff
@@ -0,0 +1,733 @@
+#
+# --------------------------------------------------------------------------------------
+
+module "monitoring-alerts-action-group-critical" {
+  source                    = "git::https://github.com/wso2/azure-terraform-modules.git//modules/azurerm/Monitor-Action-Groups?ref=v0.5.0"
+  montior_action_group_name = join("-", [var.project, "critical", var.environment, var.padding])
+  tags                      = local.default_tags
+  resource_group_name       = module.spoke-resource-group.resource_group_name
+  short_name                = "Critical"
+  email_receivers           = var.monitoring_alerts_actions_emails_critical
+  actions_webhook_critical  = []
+  depends_on                = [module.spoke-resource-group]
+}
+
+module "monitoring-alerts-action-group-warning" {
+  source                    = "git::https://github.com/wso2/azure-terraform-modules.git//modules/azurerm/Monitor-Action-Groups?ref=v0.5.0"
+  montior_action_group_name = join("-", [var.project, "warning", var.environment, var.padding])
+  tags                      = local.default_tags
+  resource_group_name       = module.spoke-resource-group.resource_group_name
+  short_name                = "Warning"
+  actions_webhook_critical  = []
+  email_receivers           = var.monitoring_alerts_actions_emails_warning
+  depends_on                = [module.spoke-resource-group]
+}
+
+
+## Service health and resource health
+module "health-monitoring-alerts" {
+  source              = "git::https://github.com/wso2/azure-terraform-modules.git//modules/azurerm/Monitor-Log-Alerts?ref=v0.1.0"
+  tags                = local.default_tags
+  resource_group_name = module.spoke-resource-group.resource_group_name
+  environment         = var.environment
+  project             = var.project
+  padding             = var.padding
+
+  recommendation_alerts = {}
+
+  resource_health_alerts = {
+    "resource_health_critical_component_alert_p1" = {
+      scopes = [
+        module.premium-fileshare-storage-account.storage_account_id,
+        module.spoke-virtual-network.virtual_network_id,
+        module.deve-key-vault.vault_id
+      ]
+      description             = "Monitoring Resource Health alerts of level Critical"
+      criteriaLevel           = "Critical"
+      criteriaCategory        = "ResourceHealth"
+      criteriaReason          = ["PlatformInitiated"]
+      currentStatuses         = ["Unavailable"]
+      previousStatuses        = ["Available", "Unknown"]
+      monitor_action_group_id = module.monitoring-alerts-action-group-critical.monitor_action_group_id
+      status                  = "Active"
+      reason                  = "[p1]resourcehealthcriticalcomponents"
+      query_enabled           = true
+      target_resource_id      = null
+      target_resource_type    = null
+      target_resource_group   = null
+    },
+    "resource_health_critical_component_alert_p3" = {
+      scopes = [
+        module.bastion.bastion_vm_id,
```

Comment:
Do we need bastion in here as we have listed it in resource_health_non_critical_component_alert_p3